src/cluster/setup.rst

```diff
@@ -45,6 +45,28 @@ possible to force an Erlang application to use a specific port rage.
 This documentation will use the range TCP ``9100-9200``. Open up those ports in
 your firewalls and it is time to test it.
 
+Configure And Test The Communication With Erlang
+================================================
+
+Make CouchDB use correct IP|FQDN and the open ports.
+----------------------------------------------------
+
+In file etc/vm.args change the the line ``-name couchdb@127.0.0.1`` to
```

Comment:
For consistency, "etc/vm.args" should be back-quoted:

> In file ``etc/vm.args`` change...

Also, you duplicated the word "the".